.github/workflows/build_and_test_debug.yml

```diff
@@ -132,6 +132,29 @@ jobs:
 
       - name: Build iOS Client
         run: npm run action cordova/build ios
+        
+  spm_debug_build:
+    name: Swift Package Debug Build
+    runs-on: macos-12
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        target: [iOS, tvOS, macOS, catalyst, watchOS]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        
+      - name: Build and Test iOS
```

Comment:
Thinking more about it, I think it's best to move the macOS tests to the macOS workflow, and the iOS test to the iOS workflow. Then we can delete this workflow.